documentation/general/program-scale-testing-requirements-draft.md

```diff
@@ -193,7 +193,7 @@ integrators to track and test the designs in a common manner.
         expect more TCP and in fact in some cases we meter UDP as a
         potential source of DoS.
 
-    -   Connection aging set to 1 sec and requires each connection or
+    -   Connection aging set to 5 sec and requires each connection or
         bi-directional flow to receive one packet every second in each
```

Comment:
Should this say "receive one packet every 5 seconds" ?

Reply:
corrected to 4.9 sec as long as the packet is received right before the 5s  mark  flow should remain active